viewer/src/culling/SectorCuller.ts

```diff
@@ -2,9 +2,13 @@
  * Copyright 2020 Cognite AS
  */
 
-import { OperatorFunction } from 'rxjs';
 import { WantedSector } from '../data/model/WantedSector';
+import { DetermineSectorsByProximityInput } from '../models/cad/determineSectors';
 
-export interface SectorCuller<Input> {
```

Comment:
@j-bjorne you wanted this to be generic. Any comments on whether we should keep it that way?